docs/kubernetes/KubernetesPolicy.md

```diff
@@ -0,0 +1,63 @@
+# Programming Calico Policy in Kubernetes
+The [Calico Kubernetes plugin](https://github.com/projectcalico/calico-docker/blob/master/docs/kubernetes/KubernetesIntegration.md) allows you to specify networking policy in the Kubernetes API using pod annotations. 
+
+## Prerequisites
+* A Kubernetes Deployment
+    - To implement service policy using recent builds of Kubernetes, make sure you have configured your kube-proxy using the `--legacy-userspace-proxy=false` option
+* [v0.1.0+](https://github.com/projectcalico/calico-kubernetes/releases) of the Calico Kubernetes Plugin
+    - For more information on how to integrate Calico into your Kubernetes Deployment, view our [integration doc](KubernetesIntegration.md)
+
+### Declaring Policy
+To enforce a policy rule to your pod, add a `policy` key to the annotations section of your pod's metadata. This `policy` key should map to a single string that outlines networking behavior semantically.
```

Comment:
"outlines networking behavior semantically" is a bit odd sounding.

"The policy should map to a single string which defines network behavior.  The syntax for this can be seen below"